client/state/data-layer/wpcom-http/index.js

```diff
@@ -21,19 +26,23 @@ import { extendAction } from 'state/utils';
 const fetcherMap = method => get( {
 	GET: wpcom.req.get.bind( wpcom.req ),
 	POST: wpcom.req.post.bind( wpcom.req ),
-}, method, noop );
+}, method, null );
 
 export const successMeta = data => ( { meta: { dataLayer: { data } } } );
 export const failureMeta = error => ( { meta: { dataLayer: { error } } } );
 export const progressMeta = ( { size: total, loaded } ) => ( { meta: { dataLayer: { progress: { total, loaded } } } } );
 
-const queueRequest = ( { dispatch }, action, next ) => {
+export const queueRequest = ( processIngress, processEgress ) => ( { dispatch }, rawAction, next ) => {
```

Comment:
could, though I chose `ingress` and `egress` after much thought on the matter. these are industry standard terms for data flowing into and out of networking devices/places. `inbound` and `outbound` may be easier for many though they are less specific.

this is not a blocker to me.